volume/store/store.go

```diff
@@ -260,7 +260,10 @@ func (s *VolumeStore) create(name, driverName string, opts, labels map[string]st
 		if v.DriverName() != driverName && driverName != "" && driverName != volume.DefaultDriverName {
 			return nil, errNameConflict
 		}
-		return v, nil
+		// check exist in driver
```

Comment:
Shouldn't this be above the "if" block above (line 260)? And the `driverName != .....` can be removed from that if (if I read correctly)